diff --git a/node_modules/sequelize/lib/sequelize.js b/node_modules/sequelize/lib/sequelize.js
index 0db3f01..b31213c 100644
--- a/node_modules/sequelize/lib/sequelize.js
+++ b/node_modules/sequelize/lib/sequelize.js
@@ -54,7 +54,8 @@ class Sequelize {
     } else if (arguments.length === 1 && typeof database === "string" || arguments.length === 2 && typeof username === "object") {
       config = {};
       options = username || {};
-      const urlParts = url.parse(arguments[0], true);
+      //const urlParts = url.parse(arguments[0], true);
+      const urlParts = url.parse(options.use_env_variable, true);
       options.dialect = urlParts.protocol.replace(/:$/, "");
       options.host = urlParts.hostname;
       if (options.dialect === "sqlite" && urlParts.pathname && !urlParts.pathname.startsWith("/:memory")) {
